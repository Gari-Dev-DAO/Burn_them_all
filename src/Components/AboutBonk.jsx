

const AboutBonk = () => {
  return (
    <div className="aboutBonk">
      <button className="nftbtn">LeaderBoard</button>
      <p style={{ color: "#14D9E5", width: "55%" }}>
        It is the first meme crypto on Solana blockchain and has been
        responsible for boosting the price of SOL token in the past weeks
      </p>
    </div>
  );
};

export default AboutBonk;
