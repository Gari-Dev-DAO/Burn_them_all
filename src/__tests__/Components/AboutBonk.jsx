import { render } from "@testing-library/react";
import AboutBonk from "../../Components/AboutBonk";

describe("AboutBonk", () => {
  it("renders component", () => {
    const { container } = render(<AboutBonk />);
    expect(container).toMatchSnapshot();
  });
});
