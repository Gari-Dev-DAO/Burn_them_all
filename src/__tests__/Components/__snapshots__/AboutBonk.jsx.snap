// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutBonk renders component 1`] = `
<div>
  <div
    class="aboutBonk"
  >
    <button
      class="nftbtn"
    >
      LeaderBoard
    </button>
    <p
      class="infoText"
    >
      It is the first meme crypto on Solana blockchain and has been responsible for boosting the price of SOL token in the past weeks
    </p>
  </div>
</div>
`;
